torch/csrc/jit/codegen/cuda/executor.cpp

```diff
@@ -389,9 +397,11 @@ LaunchParams FusionExecutor::computeLaunchParams(
                 "Cannot validate parallelization scheme, "
                 "this may be due to mixed broadcast axes that are parallelized.");
           }
-        } else {
-          // Bind the launch constraint into our evaluation context
+        } else if (!expr_eval.precomputedIntegers()) {
```

Comment:
Why wouldn't this be true?

Reply:
I was only trying to get run time latency in this PR and there're still calls to this function in `compileFusion` that will use legacy evaluator path. I could just clean up that part in the same PR as well if needed.

Reply:
Follow up seems fine. I think we should try to remove the legacy evaluators as soon as possible.